src/azure-cli/azure/cli/command_modules/appservice/tests/latest/test_webapp_commands.py

```diff
@@ -1903,35 +1903,13 @@ def test_one_deploy(self, resource_group):
 
 class DomainScenarioTest(ScenarioTest):
     @ResourceGroupPreparer(location=WINDOWS_ASP_LOCATION_WEBAPP)
-    def test_domain_e2e(self, resource_group):
+    def test_domain_create(self, resource_group):
         contacts = os.path.join(TEST_DIR, 'domain-contact.json')
-        webapp_name = "test-webapp-hostname"
-        plan_name = "webapp-plan"
-        slot = 'test'
-        # Pre-Requisite: HostNames created
-        apphostname = "adorenow.net"
-        slothostname = "managedcerts.net"
         self.cmd("appservice domain create -g {} --hostname {} --contact-info=@\'{}\' --dryrun".format(
             resource_group, "testuniquedomainname1.com", contacts)
         ).assert_with_checks([
             JMESPathCheck('agreement_keys', "['DNRA', 'DNPA']")
         ])
-        plan_id = self.cmd('appservice plan create -g {} -n {} --sku S1'.format(
-            resource_group, plan_name)).get_output_in_json()['id']
-        self.cmd('webapp create -g {} -n {} --plan {}'.format(resource_group, webapp_name, plan_id), checks=[
-            JMESPathCheck('name', webapp_name)])
-        self.cmd("webapp config hostname add -g {} --webapp-name {} --hostname {} ".format(
-            resource_group, webapp_name, apphostname)
-        ).assert_with_checks([
-            JMESPathCheck('siteName', webapp_name)
-        ])
-         #create an empty slot
-        self.cmd('webapp deployment slot create -g {} -n {} --slot {} --debug'.format(resource_group, webapp_name, slot))
-        self.cmd("webapp config hostname add -g {} --webapp-name {} --slot {} --hostname {} ".format(
-            resource_group, webapp_name, slot, slothostname)
-        ).assert_with_checks([
-            JMESPathCheck('name', "{}/{}/{}".format(webapp_name, slot, slothostname))
-        ])
```

Comment:
May I ask why did you delete these test commands?

Reply:
This test should suppose to remove in [this](https://github.com/Azure/azure-cli/pull/19348) pull request. But some how I got messed up  and missed to do. So just added here.

Reply:
OK